simplecpp.cpp

```diff
@@ -1716,66 +1716,123 @@ namespace simplecpp {
         return ostr.str();
     }
 #else
-#define realFilename(f)  f
+    // Boring implementation, but could help in platform-independent tests:
```

Comment:
As far as I know realFilename just changes the case so it matches the filesystem. Do you see any additional behaviour?